framework/include/utils/RankTwoTensor.h

```diff
@@ -102,14 +104,15 @@ class RankTwoTensorTempl : public TensorValue<T>
     general = 9
   };
 
-  /**
-   * Constructor that takes in 3 vectors and uses them to create rows
-   * _coords[0][i] = row1(i), _coords[1][i] = row2(i), _coords[2][i] = row3(i)
-   */
+  // deprecated constructor (replaced by initializeFrom)
   RankTwoTensorTempl(const TypeVector<T> & row1,
                      const TypeVector<T> & row2,
                      const TypeVector<T> & row3);
 
+  /// named constructor for initializing symetrically
```

Comment:
Making this more descriptive (explaining what those vectors passed in are) would be helpful.